script/cleanup.sh

```diff
@@ -13,6 +13,13 @@ if [ -d "/var/lib/dhcp" ]; then
     rm /var/lib/dhcp/*
 fi
 
+# Blank machine-id (DUID) so machines get unique ID generated on boot.
+# https://www.freedesktop.org/software/systemd/man/machine-id.html#Initialization
+echo "==> Blanking systemd machine-id"
```

Comment:
curious: is this a general improvement that also applies to 16.04 and 18.04, or was this specific to 20.04?

Reply:
Yes this is for all ubuntu versions.
The machine-id has to be unique since it is defined as such and some applications depend on it. This way the machine-id will be re-generated every time a new box is instantiated.